### PR TITLE
Add some details for int tests to README

### PR DESCRIPTION
**Pull request by zeisss**
For me the readme reagarding the integration tests was a bit unclear. I extended it a bit after consulting Nesurion.

ping @Nesurion 
ping @joe grammar check please :P

### CODE REVIEW

**Comment by Nesurion** [comment]
LGTM

**Comment by JosephSalisbury** [comment]
you know @joe isn't me right

you probably want @JosephSalisbury :gb:

File: README.md [hunk 1]
```diff
@@ -21,14 +21,40 @@ inagoctl -h
 
 ## Running integration tests
 
-Runnning `make int-test` will execute the integration test suit. The tests need
-to run against fleet. Starting and destroying the test machine is done via the
-make target `int-test`, so you don't need to start a machine yourself.
+Runnning `make int-test` will execute the integration test suit in a docker container.
```
Comment (JosephSalisbury) [inline README.md]: _suite_, not suit

Comment (JosephSalisbury) [inline README.md]: _Docker_, not docker

Comment (JosephSalisbury) [inline README.md]: Actually, ignore that last one, we'll be here all day.

File: README.md [hunk 2]
```diff
@@ -21,14 +21,40 @@ inagoctl -h
...
+The tests need to run against fleet, so we manage a vagrant box. Starting and destroying
+this test machine is done via the make target `int-test`.
 
 ### Integration Test Machine Configuration
 
-Set the `FLEET_ENDPOINT` environment variable to IP listed under the vboxnet
-interface of your docker-machine.
+Since the integration tests run in your docker machine (not the fleet machine), we need to
+provide it with an IP. We use portfowarding on your host for this.
```
Comment (JosephSalisbury) [inline README.md]: _port forwarding_, two words

File: README.md [hunk 3]
```diff
@@ -21,14 +21,40 @@ inagoctl -h
...
+	ether 0a:00:27:00:00:04
+vboxnet5: flags=8843<UP,BROADCAST,RUNNING,SIMPLEX,MULTICAST> mtu 1500
+	ether 0a:00:27:00:00:05
+	inet 172.17.8.1 netmask 0xffffff00 broadcast 172.17.8.255
+vboxnet6: flags=8843<UP,BROADCAST,RUNNING,SIMPLEX,MULTICAST> mtu 1500
+	ether 0a:00:27:00:00:06
+	inet 192.168.59.3 netmask 0xffffff00 broadcast 192.168.59.255
+```
+
+On Stephan's machine the docker machine is in the `vboxnet6` network and the coreos VMs
```
Comment (JosephSalisbury) [inline README.md:56]: points for correct use of possessive apostrophe

**Pull request merged**